### PR TITLE
Replace insert with upsert for import

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

**Comment by scottaubrey** [comment]
The error in the tests is exposing an issue with the way we are mocking S3 I think. The stream for each file is created in the `beforeEach()`, but now that the tests are actually trying to re-read the files, the stream is empty the second time around.

Might be one for @will-byrne or @E-Kemp to take a look at refactoring those mocked bits

File: src/controller/import-controller.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export const importController = (repo: ArticleRepository) => {
       if (results.length === 0) {
         res.send({
           status: false,
-          message: 'No new files were imported',
+          message: 'No files were imported',
         });
       } else if (results.every((value) => value)) {
```
[the remark this answers is not in the document]
Comment (scottaubrey) [reply src/controller/import-controller.ts:20]: 0 files in the bucket

[the remark this answers is not in the document]
Comment (scottaubrey) [reply src/controller/import-controller.ts:20]: Of course! 🤦  

It means all results are true - so all the manuscripts resulted in a changed rows.

[the remark this answers is not in the document]
Comment (nlisgo) [reply src/controller/import-controller.ts:20]: In which case, to we have expect the `else` condition to be met resulting in `Some new items imported`? Or should we refactor to remove this check?

**Review by scottaubrey — APPROVED** [review]

**Pull request merged**